net/src/memory_worker.rs

```diff
@@ -183,316 +183,4 @@ mod tests {
 
         memory_worker_1.tick().unwrap();
     }
-
-    #[test]
-    #[cfg_attr(tarpaulin, skip)]
-    fn can_memory_network_flow() {
```

Comment:
I absolutely understand wanting to take these out so we don't have to maintain two sets of basically duplicated unit tests.

Since `test_bin` is not yet in CI, though, I worry about unrelated work accidentally breaking networking if CI is not covering it.

What do you think about adding a unit `#[test]` to the `test_bin` crate that executes the full suite, but only for the "In-Memory" module, as a baby step to getting this in CI?

Reply:
good idea! Let me try that.